### PR TITLE
Support custom id key per model

### PR DESCRIPTION
**Pull request by santib**

### CODE REVIEW

**Review by juanmanuelramallo — APPROVED** [review]

File: spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb [hunk 1]
```diff
@@ -329,6 +329,12 @@ def up
     add_index :bar_habtms_foo_habtms, [:foo_habtm_id]
     add_index :bar_habtms_foo_habtms, [:bar_habtm_id]
 
+    create_table :custom_id_key_records, force: true do |t|
+      t.column :uuid, :string
```
Comment (sebaherrera07) [inline spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb:333]: What about adding `, primary_key: true` here?

Comment (santib) [reply spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb:333]: but the idea is that it isn't the primary_key. I want this model to have the Rails id key as the primary and the uuid, that would be the case for our HerokuConnect models (id and sfid)

Comment (sebaherrera07) [reply spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb:333]: Good I understand. Then maybe adding an index may be useful?

Comment (santib) [reply spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb:333]: But this is just for the dummy app used for testing, check that it's in `spec/dummy_app/...`

**Review by sebaherrera07 — APPROVED** [review]

**Pull request merged**